packages/sdk/src/oracle/constants.ts

```diff
@@ -0,0 +1,29 @@
+import { ethers } from "ethers";
+
+export const MULTICALL2_ADDRESS = "0x5ba1e12693dc8f9c48aad8770482f4739beed696";
```

Comment:
I imagine in the future this will need to depend on the chain id (when we'll deploy OO on other chains)

Reply:
good call, yea i can move that now